nubis/puppet/files/nubis_users/nubis-users.toml

```diff
@@ -0,0 +1,12 @@
+[template]
+src = "nubis-users.tmpl"
+dest = "/etc/puppet/yaml/nubis_users/common.yaml"
+prefix = "/environment/nubis-users"
```

Comment:
> AFAIK, we'll have 3 Lambda functions in each VPCs, pushing to Consul, so each of them pushing to a different prefix seems like just like different arguments fed via the CloudWatch alarm, right ?

At this moment no its not an argument feed to CloudWatch its just a config file option, we can probably add it as a cloudwatch alarm and it shouldn't be too hard but right at this moment its just a config file option